src/chat/zulip.md

```diff
@@ -0,0 +1,84 @@
+## Zulip
+
+[Rust's Zulip](rust-lang.zulipchat.com) is used by a number of teams, notably the compiler and language
+teams, along with their working groups.
+
+Zulip can be an unintuitive platform to get started with. The UI is divided into
+two areas: a list of streams on the left, and the main chat in the middle.
+
+## Streams
+
+These are similar to "channels" on other platforms (i.e., there should not be
+too many). On the other hand, you can choose which streams you subscribe to, so
+there can be more than channels on other platforms.
+
+Streams are appropriate for any Rust official group. For example, working
+groups, project groups, teams are all examples of official groups. These should
+ideally also be represented in the [team repository](../infra/team-maintenance.md).
+
+A stream should be named such as `#t-{team}/{group name}`. For example,
+`#t-compiler/wg-parallel-rustc`.
+
+If no top-level team exists, or the group spans multiple teams (e.g.,
+project-ffi-unwind), then the top level team should be omitted.
+
+It is not generally appropriate to have groups that do not correspond to a set
+of people. For example, `#t-compiler/help` would not be appropriate, as such
+discussion should go into `#t-compiler` (the catch-all compiler stream).
```

Comment:
I don't agree here -- I think it makes sense to have streams that correspond to topic areas. I find `t-compiler` a bit hard to follow as is. Some "topic areas" I think we probably want:

- help (pre-existing)
- llvm
- the cargo-bisect-rustc project

The latter could be construed as a set of people, but it's not totally clear.

Reply:
I agree with Niko here. There's also help-wanted which I think could help pass ideas from contributors to people that want to start contributing.